libs/framework/src/framework.c

```diff
@@ -2307,16 +2317,6 @@ static celix_status_t framework_loadLibrary(framework_pt framework, const char *
         if (*handle == NULL) {
             error = celix_libloader_getLastError();
             status =  CELIX_BUNDLE_EXCEPTION;
-        } else {
```

Comment:
Removing this and moving this to loadLibraries (and only for the activator libraries) means that not all libraries mentioned in Private-Library and Import-Library are added to the handles list. 
I think this is not the desired result. 

I expected this issues to be solved during the stopping if a bundle (i.e. during the closing of a library).

Could you first add an unit test to reproduce this issue. We already have some test for starting/stopping bundles. See
https://github.com/apache/celix/blob/40e33daaeb12f3bc38d19a477a861ffffe72b734/libs/framework/gtest/src/bundle_context_bundles_tests.cpp#L38
https://github.com/apache/celix/blob/40e33daaeb12f3bc38d19a477a861ffffe72b734/libs/framework/gtest/subdir/CMakeLists.txt#L18
https://github.com/apache/celix/blob/40e33daaeb12f3bc38d19a477a861ffffe72b734/libs/framework/gtest/CMakeLists.txt#L18